damus/Models/UserSettingsStore.swift

```diff
@@ -227,6 +227,19 @@ class UserSettingsStore: ObservableObject {
             return deepl_api_key != ""
         }
     }
+
+    func delete_settings(_ pubkey: String) throws {
```

Comment:
There's some danger here where if we introduce new settings, we need to remember to add it here. Not sure what the best way to mitigate that risk would be. But this is better than what we have currently.